tools/msvs/msi/i18n/en-us.wxl

```diff
@@ -14,7 +14,7 @@
     <String Id="NativeToolsDlgBannerBitmap">WixUI_Bmp_Banner</String>
     <String Id="NativeToolsDlgIntro">Some npm modules need to be compiled from C/C++ when installing. If you want to be able to install such modules, some tools (Python 2 and Visual Studio Build Tools) need to be installed.</String>
     <String Id="NativeToolsDlgInstallCheckbox">Automatically install the necessary tools. The script will pop-up in a new window after the installation completes.</String>
-    <String Id="NativeToolsDlgManualDetails">Alternatively, follow the instructions at <![CDATA[<a href="https://github.com/nodejs/node-gyp#on-windows">https://github.com/nodejs/node-gyp#on-windows</a>]]> to install the dependencies yourself.</String>
+    <String Id="NativeToolsDlgManualDetails">Note that this will also install Boxstarter and Chocolatey. Alternatively, follow the instructions at <![CDATA[<a href="https://github.com/nodejs/node-gyp#on-windows">https://github.com/nodejs/node-gyp#on-windows</a>]]> to install the dependencies yourself.</String>
```

Comment:
This should go at the end of the line above, and actually fits there nicely. It makes sense to me to keep the line above (that goes next to the checkbox) about automatic installation and this one about pointing users to instructions to install manually.